app/Resources/assets/css/application.scss

```diff
@@ -31,7 +31,16 @@ body {
 }
 
 .table th {
-    font-weight: normal;
+    font-weight: bold;
+    max-width: 250px;
+}
+
+.table td {
+    max-width: 250px;
+}
```

Comment:
Here we could instead make every column except the first have `white-space: nowrap`, then do away with the `max-width`. That way larger screens can take advantage of the extra space and not wrap at 250px. For this I'd use the `text-nowrap` class on all but the first `<th>` element.

Reply:
Isn't that default? It won't wrap and I'm trying to do that. If I do away with max-width and add text-nowrap class to columns besides the first, it'd do nothing effectively. How can I tell it to wrap after a limit without specifying max-width for columns? I could do the max-width in % if you think that's better?

Reply:
If `text-nowrap` is on all the other cells (`<td>` and `<th>`), then the title should wrap if there's not enough room, looking similar to your second screenshot. I suppose you might run into a scenario where the headings are really close together (though you could add padding), depends on what you want. Just a recommendation, the `max-width` will indeed make sure things don't look bad so I've no problem with it :)

Reply:
I tried this. It didn't wrap but overflowed outside the viewport. 
<img width="1440" alt="screen shot 2018-01-10 at 10 46 03 am" src="https://user-images.githubusercontent.com/5735280/34756827-7c9e8474-f5f3-11e7-88c9-e5091fecdf52.png">

Gonna push the code now so you can try it too.

I know `max-width` isn't a really neat solution but I couldn't find any other that'd work well.